cmd/systrack/main.go

```diff
@@ -3,49 +3,48 @@ package main
 import (
 	"log"
 
+	"go.mozilla.org/systrack"
+
 	"github.com/mozilla/scribe"
 	"github.com/sirupsen/logrus"
 	"go.mozilla.org/mozlogrus"
 )
 
 func main() {
+	var (
+		p   systrack.PkgMsg
+		err error
+	)
 	hook, err := mozlogrus.NewKinesisHook("secops-dumper")
 	logger := logrus.New()
 	logger.Hooks.Add(hook)
-	logger.Formatter = &mozlogrus.MozLogFormatter{LoggerName: "secops-dumper", Type: "app.log"}
+	logger.Formatter = &mozlogrus.MozLogFormatter{LoggerName: "systrack", Type: "pkg"}
```

Comment:
The idea is to use `Type` to switch the parsing in `systrack-lambda` to the corresponding message type.